tests/pc/test_lag_2.py

```diff
@@ -169,28 +178,25 @@ def run_single_lag_lacp_rate_test(self, lag_name):
                     logger.info("Changing lacp rate to slow for %s in %s" % (neighbor_lag_member, peer_device))
                     vm_host.set_interface_lacp_rate_mode(neighbor_lag_member, 'normal')
 
-    def run_single_lag_test(self, lag_name):
+    def run_single_lag_test(self, lag_name, lag_facts):
```

Comment:
Changes are good, minor comment instead of passing lag_facts to each function .. could pass it when we create LagTest object and store it during init as self.lag_facts.

Reply:
@judyjoseph lag_facts changes as part of test cases since we do changes on neighbor lag config and get it again.
so we can't store one time.

Reply:
@judyjoseph can you review and approve.